sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionEventArgs.cs

```diff
@@ -139,7 +139,8 @@ public class ProcessSessionEventArgs : EventArgs
         /// </summary>
         public virtual void ReleaseSession() =>
             // manager will be null if instance created using the public constructor which is exposed for testing purposes
-            _manager?.CancelSession();
+            // This will be awaited when closing the receiver.
+            _ = _manager?.CancelSessionAsync();
```

Comment:
Should we consider attaching a `ContinueWith` scoped to exceptions to capture any problems in the logs?